<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cp2kInputDialog</class>
 <widget class="QDialog" name="Cp2kInputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CP2K Input</string>
  </property>
  <layout class="QGridLayout">
   <item row="6" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="resetAllButton">
       <property name="text">
        <string>Reset All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="defaultsButton">
       <property name="text">
        <string>Defaults</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="computeButton">
       <property name="text">
        <string>Submit Calculation...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="generateButton">
       <property name="text">
        <string>Save File...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QTextEdit" name="previewText">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="modeTab">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="basicWidget">
      <attribute name="title">
       <string>&amp;Basic Setup</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Title:</string>
         </property>
         <property name="buddy">
          <cstring>titleEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="titleEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Filename Base:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="baseNameEdit">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>job</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Calculate:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>calculateCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QComboBox" name="calculateCombo"/>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Method:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>methodCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QComboBox" name="methodCombo"/>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Basis set:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>basisCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QComboBox" name="basisCombo"/>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Functional</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="functionalCombo">
           <property name="accessibleDescription">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>16</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>MM</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="emaxSplineSpin_label">
         <property name="text">
          <string>FF Emax Spline Spin</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QDoubleSpinBox" name="emaxSplineSpin">
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="minimum">
            <double>-99.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.000010000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>FF RCUT NB</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QDoubleSpinBox" name="rcutnbSplineSpin">
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="minimum">
            <double>-99.000000000000000</double>
           </property>
           <property name="value">
            <double>-1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Poisson EWALD type</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QComboBox" name="ewaldtypeCombo"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Poisson EWALD Alpha</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QDoubleSpinBox" name="ewaldalphaSpin">
           <property name="minimum">
            <double>-99.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Poiosson EWALD GMAX</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QSpinBox" name="ewaldgmaxSpin">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>QM</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="LSD_label">
         <property name="text">
          <string>LSD</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_28">
         <item>
          <widget class="QCheckBox" name="lsdcheckBox">
           <property name="text">
            <string>TRUE</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_26">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>MAX SCF</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_29">
         <item>
          <widget class="QSpinBox" name="maxscfspinBox">
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="value">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_27">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>EPS SCF</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_25">
         <item>
          <widget class="QDoubleSpinBox" name="epsscfSpinBox">
           <property name="decimals">
            <number>8</number>
           </property>
           <property name="value">
            <double>0.000010000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_23">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>SCF GUESS</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_26">
         <item>
          <widget class="QComboBox" name="scfguessComboBox"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_24">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>OUTER MAX SCF</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_27">
         <item>
          <widget class="QSpinBox" name="outerMaxscfSpinBox">
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="value">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_25">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>OUTER EPS SCF</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_30">
         <item>
          <widget class="QDoubleSpinBox" name="outerEpsscfSpinBox">
           <property name="decimals">
            <number>8</number>
           </property>
           <property name="value">
            <double>0.000010000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_28">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>OT T MINIMIZER</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_31">
         <item>
          <widget class="QComboBox" name="otminimizerComboBox"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_29">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>calculateCombo</tabstop>
  <tabstop>basisCombo</tabstop>
  <tabstop>functionalCombo</tabstop>
  <tabstop>methodCombo</tabstop>
  <tabstop>previewText</tabstop>
  <tabstop>resetAllButton</tabstop>
  <tabstop>defaultsButton</tabstop>
  <tabstop>generateButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
